torch/_inductor/compile_fx.py

```diff
@@ -590,8 +516,11 @@ def fx_codegen_and_compile(
         post_grad_graphs_log.info("%s", lazy_format_graph_code("AFTER POST GRAD", gm))
 
     with V.set_fake_mode(fake_mode):
+        # example_inputs will be used by AOTInductor to dry-run the generated code for Triton kernel tuning,
+        # and for the forward pass, we have the real inputs to be used as example_inputs.
         graph = GraphLowering(
             gm,
+            example_inputs=V.real_inputs if is_inference else example_inputs,
```

Comment:
nit - Looks like for the non-forward pass, ```example_inputs``` are faked tensors or sym values. Could you add a comment to make it clear? Sometimes, it's hard to tell if an example_input is of FakedTensor or not by reading the code. Thanks.